Configuration/TypoScript/constants.txt

```diff
@@ -5,6 +5,8 @@ themes.configuration.siteName = t3kit
 themes.languages.default.isoCode = en_GB
 themes.configuration.headerComment = This website is based on t3kit project by Pixelant - www.t3kit.com
 
+# Change max width image from 600 to 900
```

Comment:
@Chanpiseth09 I think this comment is unnecessary. Comments should not be a change log. This would be more useful in your commit description.